core/chains/evm/client/rpc_client.go

```diff
@@ -33,6 +34,47 @@ import (
 	ubig "github.com/smartcontractkit/chainlink/v2/core/chains/evm/utils/big"
 )
 
+//go:generate mockery --quiet --name ChainClientRPC --structname MockChainClientRPC --filename "mock_chain_client_rpc_test.go" --inpackage --case=underscore
```

Comment:
The MockChainClientRPC is not used. Do we need it?

Reply:
It's being used for `TestChainClient_BatchCallContext` inside of `chain_client_test.go`

Reply:
We no longer need it as I will find another way to test `BatchCallContext`.